modules/nlu/src/views/full/entities/PatternEntity.tsx

```diff
@@ -30,6 +30,12 @@ export const PatternEntityEditor: React.FC<Props> = props => {
   const [examplesStr, setExampleStr] = useState((props.entity.examples || []).join('\n'))
   const [allExamplesMatch, setExamplesMatch] = useState<boolean>(true)
 
+  const getEntityId = (entityName: string) =>
```

Comment:
That function is duplicated in `modules/nlu/src/views/full/entities/ListEntity.tsx`

Reply:
I know, I wanted to ship this real fast, I have an actual real fix coming in 12.8